docs/source/testing/android-studio-plugin.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Android Studio / IntelliJ plugin
+---
+
+To help your day-to-day development with Apollo Kotlin, a plugin for Android Studio and IntelliJ is available.
+
+## Installation
+
+The plugin is available on the JetBrains plugin repository. To install it from the IDE, go to
+<kbd>Settings</kbd> | <kbd>Plugins</kbd> | <kbd>Marketplace</kbd> | search for "Apollo GraphQL" | <kbd>Install</kbd>
+
+The plugin is supported on:
+
+- IntelliJ 2022.2 and above
+- Android Studio 2022.2 (Flamingo) and above
```

Comment:
This is a recipe for stale information as this is changing fast. I would rephrase with something like so at least to have some failsafe:

```suggestion
At the time of writing, the plugin is supported on:

- IntelliJ 2022.2 and above
- Android Studio 2022.2 (Flamingo) and above

Please check the [plugin page](https://plugins.jetbrains.com/plugin/20645-apollo-graphql) for up to date information.
```